src/App.tsx

```diff
@@ -124,12 +129,25 @@ function App() {
     }
   };
 
+  const setExtensionIdToLocalStorage = useCallback(
+    (_extensionId: string): void => {
```

Comment:
generally we try to use `_` prefix to communicate that a param is unused. Instead can we rename this to something like `newExtensionId` maybe?

Reply:
resolved in [de81e](https://github.com/MetaMask/test-dapp-multichain/pull/22/commits/de81e7e392022e41ac8edd4066a44826540fb4d5)